test-cypress/integration/explore/class.hierarchy.spec.js

```diff
@@ -10,13 +10,18 @@ const CLASS_HIERARCHY = 'class hierarchy';
 describe('Class hierarchy screen validation', () => {
     let repositoryId;
 
-    beforeEach(() => {
+    before(() => {
```

Comment:
I don't see why you changed `beforeEach` and `afterEach` hooks here and in the rest of the tests?